src/lib/elements/forms/inputTextarea.svelte

```diff
@@ -53,8 +55,33 @@
             bind:value
             bind:this={element}
             on:invalid={handleInvalid} />
+        {#if maxlength}
+            <span class="text-counter" data-active={!!value}>
+                <span class="text-counter-count">{value?.length ?? 0}</span>
+                <span class="text-counter-separator" />
+                <span class="text-counter-max">{maxlength}</span>
+            </span>
+        {/if}
     </div>
     {#if error}
         <Helper type="warning">{error}</Helper>
     {/if}
 </FormItem>
+
+<style class="scss">
+    .text-counter {
+        --p-text-counter-color: var(--color-neutral-50);
+    }
+
+    .text-counter[data-active='true'] {
+        --p-text-counter-color: var(--color-neutral-70);
+    }
```

Comment:
wouldn't be adding a class for this more elegant?

Reply:
Do you mean adding a class in Pink? If not, IMO data-attributes make more sense than classes for controlling element 'states', but it's personal preference only, I can add a class if you prefer.

Reply:
we can just add classes in the svelte component here 👍🏻